dlib/common/include/text.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
with a little googling....it seems to be a git "convention" to the `touch .gitkeep` within that folder